### PR TITLE
Rb starting on UI

### PR DESCRIPTION
**Pull request by dsimp**

### CODE REVIEW

File: app/models/like.rb [hunk 1]
```diff
@@ -19,7 +19,7 @@
 #  fk_rails_...  (photo_id => photos.id)
 #
 class Like < ApplicationRecord
-  belongs_to :fan, class_name: "User", counter_cahe: true
+  belongs_to :fan, class_name: "User", counter_cache: true
```
Comment (aldoportillo) [inline app/models/like.rb:22]: good fix

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -6,11 +6,24 @@
...
     <%= csp_meta_tag %>
 
+     <%= render partial: "shared/cdn_assets" %>
+
     <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
     <%= javascript_importmap_tags %>
   </head>
 
   <body>
+    <%= render partial: "shared/navbar" %>
```
Comment (aldoportillo) [inline app/views/layouts/application.html.erb:16]: nice formatting

File: app/views/shared/_navbar.html.erb [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        </li>
+
+         <% else %>
+        <li class="nav-item">
+          <%= link_to "Sign in", new_user_session_path, class: "nav-link" %>
+        </li>
+        <li class="nav-item">
+          <%= link_to "Sign up", new_user_registration_path, class: "nav-link" %>
+        </li>
+                <% end %>
```
Comment (aldoportillo) [inline app/views/shared/_navbar.html.erb]: fix the formatting here

File: lib/tasks/dev.rake [hunk 4]
```diff
@@ -1,3 +1,21 @@
 desc "Fill the database tables with some sample data"
-task({ :sample_data => :environment }) do
```
Comment (aldoportillo) [inline lib/tasks/dev.rake:2]: this can stay as `task({ :sample_data => :environment }) do`

**Review by aldoportillo — CHANGES_REQUESTED** [review]
Good job just fix formatting

**Review by bibimbop123 — APPROVED** [review]

File: app/views/shared/_navbar.html.erb [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+          <li class="nav-item">
+            <%= link_to "Sign up", new_user_registration_path, class: "nav-link" %>
+          </li>
+        <% end %>
+
+      </ul>
+    </div>
+  </div>
+</div>
+</div>
```
Comment (aldoportillo) [inline app/views/shared/_navbar.html.erb]: do you need this extra div?

**Review by aldoportillo — APPROVED** [review]
good job

**Pull request merged**